package.json

```diff
@@ -39,7 +39,10 @@
       "baseURL": "src",
       "lib": "src/classmentors"
     },
-    "dependencies": {},
+    "dependencies": {
+      "c3": "npm:c3@^0.4.11",
+      "d3": "npm:d3@^4.2.2"
```

Comment:
It's is:

https://github.com/c3js/c3/blob/master/package.json#L24

JSPM is installing two versions. Your import of d3 returns d3@4 but c3 is given d3@3.

You can use `jspm inspect npm:d3` to check.